.gitignore

```diff
@@ -1 +1,3 @@
 elpa
+.tramp_history
```

Comment:
This is not required, you could either ignore it in your .git/config/exclude or globally.